index.html

```diff
@@ -8,7 +8,7 @@ <h2>Code for San José</h2>
           <blockquote>
             <p>Where makers, designers, developers, and subject matter experts come together to use tech to solve civic problems.</p>
             <p><a class="btn btn-primary btn-lg" href="{{ "/docs/home/" | prepend: site.baseurl }}" role="button">Learn more</a></p>
-    </di      
+    </di
```

Comment:
Typo here? Looks like invalid HTML.

Reply:
I just removed it! Don't know how that happened.